src/test/java/mops/businesslogic/PermissionServiceTest.java

```diff
@@ -0,0 +1,74 @@
+package mops.businesslogic;
+
+import mops.exception.MopsException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Set;
+
+import static mops.businesslogic.PermissionServiceImpl.URL;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class PermissionServiceTest {
+
+    private PermissionService permissionService;
+
+    @Mock
+    private RestTemplate restTemplate;
+    private long groupId;
+    private String userName;
+
+    @BeforeEach
+    void setUp() {
```

Comment:
```suggestion
    void setup() {
```

> setup()

instead of 

> setUp()

 is used in all other tests (except 1, adding todo)